.swiftlint.yml

```diff
@@ -1,43 +1,109 @@
+excluded:
+  - ${PWD}/Carthage
+  - ${PWD}/Pods
+  - ${PWD}/DerivedData
+  - ${PWD}/.build
+  - ${PWD}/Demos/PerfectDemo/.build
+  - ${PWD}/Demos/VaporDemo/.build
```

Comment:
Unfortunately the swiftlint do not support wildcards for this use case :/

Reply:
Do you mean we have to specify file by file ?

Reply:
Not the "file" itself, but the directory, for example for `.build` we can't just use: ` - ${PWD}/**/.build/"` :(